bundles/org.openhab.binding.dominoswiss/src/main/java/org/openhab/binding/dominoswiss/internal/BlindHandler.java

```diff
@@ -0,0 +1,176 @@
+                }
+                break;
+            case CHANNEL_CONTINOUSUP:
+                if (command instanceof Number) {
+                    dominoswissHandler.continuousUp(id);
+                }
+                break;
+            case CHANNEL_CONTINOUSDOWN:
+                if (command instanceof Number) {
+                    dominoswissHandler.continuousDown(id);
+                }
+                break;
+            case CHANNEL_STOP:
+                if (command instanceof Number) {
+                    dominoswissHandler.stop(id);
+                }
+                break;
+            case UP:
+                if (command instanceof Number) {
+                    dominoswissHandler.continuousUp(id);
+                }
+                break;
+            case DOWN:
+                if (command instanceof Number) {
+                    dominoswissHandler.continuousDown(id);
+                }
+                break;
+            case SHUTTER:
+                if (command.toFullString() == DOWN) {
+                    dominoswissHandler.continuousDown(id);
+                } else if (command.toFullString() == UP) {
+                    dominoswissHandler.continuousUp(id);
+                } else if (command.toFullString() == CHANNEL_STOP) {
+                    dominoswissHandler.stop(id);
+                } else {
+                    logger.debug("Blind got command but nothing executed: {}  and ChannelUID: {}",
+                            command.toFullString(), channelUID.getIdWithoutGroup());
+                }
+
+            case TILTDOWN:
+                if (command instanceof Number) {
+                    try {
+                        dominoswissHandler.tiltDown(id);
+                    } catch (InterruptedException e) {
+                        logger.error("EGate tiltDown error: {} ", e.toString());
+                    }
+                }
+                break;
+
+            case TILTUP:
+                if (command instanceof Number) {
+                    try {
+                        dominoswissHandler.tiltUp(id);
+                    } catch (InterruptedException e) {
+                        logger.error("EGate tiltUP error: {} ", e.toString());
+                    }
+                }
+                break;
+
+            case SHUTTERTILT:
+                if (command.toFullString() == UP) {
+                    dominoswissHandler.pulseUp(id);
+                } else if (command.toFullString() == DOWN) {
+                    dominoswissHandler.pulseDown(id);
+                } else if (command.toFullString() == CHANNEL_STOP) {
+                    dominoswissHandler.stop(id);
+                } else {
+                    logger.debug("Blind got command but nothing executed: {}  and ChannelUID: {}",
+                            command.toFullString(), channelUID.getIdWithoutGroup());
+                }
+
+            default:
+                break;
+        }
+    }
+
+    @SuppressWarnings("null")
```

Comment:
Suppressions should only be used when the compiler warning is not adequate. In this case you could add null checks.